components/gitpod-db/package.json

```diff
@@ -3,7 +3,7 @@
   "license": "AGPL-3.0",
   "version": "0.1.5",
   "scripts": {
-    "build": "yarn lint && npx tsc",
+    "build": "yarn lint && npx tsc && yarn generate-latest-migration",
```

Comment:
we need to flip this around `yarn lint && yarn generate-latest-migration && npx tsc` as we are calling yarn build with `-w` flag to run tsc in watch mode.

Reply:
Can we just write a short bash script that creates the file? It seems a bit too involving otherwise

Reply:
> Can we just write a short bash script that creates the file? It seems a bit too involving otherwise

Addressed 🙏